tests/safety/common.py

```diff
@@ -474,13 +478,19 @@ def test_alternative_experience_no_disengage_on_gas(self):
     self._rx(self._user_gas_msg(0))
     self.assertTrue(self.safety.get_longitudinal_allowed())
 
-  def test_prev_brake(self):
-    self.assertFalse(self.safety.get_brake_pressed_prev())
-    for pressed in [True, False]:
-      self._rx(self._user_brake_msg(not pressed))
-      self.assertEqual(not pressed, self.safety.get_brake_pressed_prev())
-      self._rx(self._user_brake_msg(pressed))
-      self.assertEqual(pressed, self.safety.get_brake_pressed_prev())
+  def test_prev_brake(self, test="brake"):
+    def test_prev_brake(_user_brake_msg, get_brake_pressed_prev):
+      self.assertFalse(self.safety.get_brake_pressed_prev())
+      for pressed in [True, False]:
+        self._rx(_user_brake_msg(not pressed))
+        self.assertEqual(not pressed, get_brake_pressed_prev())
+        self._rx(_user_brake_msg(pressed))
+        self.assertEqual(pressed, get_brake_pressed_prev())
+
+    if test == "brake":
+      test_prev_brake(self._user_brake_msg, self.safety.get_brake_pressed_prev)
+    else:  # test with regen message and global variable
```

Comment:
this comment shouldn't be necessary. it would be more self-explanatory if the function had a better name